android/PhysicalWeb/app/src/main/res/values/config-strings.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="url_getting_started">https://google.github.io/physical-web/mobile/android/getting-started.html</string>
+    <string name="pws_endpoint_setting_key">pws_endpoint</string>
+    <string name="custom_pws_endpoint_key">custom_pws_endpoint</string>
+    <string name="custom_pws_url_key">custom_pws_url</string>
+    <string name="custom_pws_version_key">custom_pws_version</string>
+    <string name="custom_pws_api_key_key">custom_pws_key</string>
+    <string name="scan_settings_key">scan_settings</string>
+    <string name="mDNS_key">mDNS</string>
+    <string name="main_prefs_key">physical_web_preferences</string>
+    <!-- The following is a more appropriate name, but would require special code to migrate users -->
+    <!-- string name="main_prefs_key">org.physical_web.physicalweb.MAIN_PREFS</string -->
+    <string name="discovery_service_prefs_key">org.physical_web.physicalweb.DISCOVERY_SERVICE_PREFS</string>
```

Comment:
I believe because it is being used for a shared preference file and those need to be unique among all apps.